dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature

```diff
@@ -3,6 +3,27 @@
 @no_mobile
 
 Feature: Using the Lesson Edit Page
+  Scenario: Save changes using the lesson edit page for lesson without lesson plan
+    Given I create a levelbuilder named "Levi"
+    And I create a temp migrated script and lesson
```

Comment:
nice updates to UI tests! nit: change to "script and lessons" or "script with lessons"?